source/constructs/lib/common-resources/custom-resources/custom-resource-construct.ts

```diff
@@ -178,23 +176,12 @@ export class CustomResourcesConstruct extends Construct {
   }
 
   public setupCopyWebsiteCustomResource(props: SetupCopyWebsiteCustomResourceProps) {
-    // Allows the custom resource to read the static assets for the front-end from the source code bucket
-    this.sourceCodeBucket.grantRead(this.customResourceLambda, `${this.sourceCodeKeyPrefix}/*`);
-
-    this.createCustomResource(
-      "CopyWebsite",
-      this.customResourceLambda,
-      {
-        CustomAction: "copyS3assets",
-        Region: Aws.REGION,
-        ManifestKey: [this.sourceCodeKeyPrefix, "demo-ui-manifest.json"].join("/"),
-        SourceS3Bucket: this.sourceCodeBucket.bucketName,
-        SourceS3key: [this.sourceCodeKeyPrefix, "demo-ui"].join("/"),
-        DestS3Bucket: props.hostingBucket.bucketName,
-        Version: this.solutionVersion,
-      },
-      this.conditions.deployUICondition
-    );
+    // Stage static assets for the front-end from the local
+    /* eslint-disable no-new */
```

Comment:
might as well disable this rule project-wide, it conflicts with cdk's design

Reply:
that will hide legit warnings